### PR TITLE
Missing timestamps

### PR DESCRIPTION
**Pull request by femalves**

### CODE REVIEW

File: biblib/views/library_view.py [hunk 1]
```diff
@@ -175,11 +174,11 @@ def solr_update_library(cls, library_id, solr_docs, session):
...
             else:
                 new_library_bibcodes[bibcode] = library.bibcode[bibcode]
-        session.add(library)
-        session.commit()
-        if updates['update_list']:                
-            cls.update_database(session, library, new_library_bibcodes, updates)
-
+        
+        library.bibcode = new_library_bibcodes
+        cls.update_library(session, library)
```
Comment (tjacovich) [inline biblib/views/library_view.py:179]: This modification looks good. The only change I would make would be to add some sort of `update` flag that is set to `True` if any of the above things are changed. If we don't do that, I think `sqlalchemy_continuum` will capture a lot of duplicate library updates as we are explicitly marking the library as modified every time we pass it to the `update_library` method.

**Review by tjacovich — CHANGES_REQUESTED** [review]
Looks good. Just one update we might want to make so we don't end up with a lot of duplicate entries in the versions table for the libraries.

File: biblib/views/library_view.py (hunk 1 above)
Comment (femalves) [reply biblib/views/library_view.py]: I see what you mean. I just updated and put the flags back.

**Review by tjacovich — CHANGES_REQUESTED** [review]
I think we're nearly there. One last comment on the update logic.

File: biblib/views/library_view.py [hunk 2]
```diff
@@ -175,11 +176,14 @@ def solr_update_library(cls, library_id, solr_docs, session):
...
                 new_library_bibcodes[bibcode] = library.bibcode[bibcode]
-        session.add(library)
-        session.commit()
-        if updates['update_list']:                
-            cls.update_database(session, library, new_library_bibcodes, updates)
-
+        
+        if updated_timestamp: 
+            cls.update_library(session, library)
+        if updates['update_list']:
```
[the remark this answers is not in the document]
Comment (femalves) [reply biblib/views/library_view.py:180]: I appreciate your patience

**Review by tjacovich — APPROVED** [review]
Looks good. Feel free to merge when ready.

**Pull request merged**